metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -87,6 +100,21 @@ class LDAPSourceConfig(ConfigModel):
         default=20, description="Size of each page to fetch when extracting metadata."
     )
 
+    # default mapping for attrs
+    attrs_mapping: Dict[str, Any] = {}
+
+
+def guess_person_ldap(attrs: Dict[str, Any], config: LDAPSourceConfig) -> Optional[str]:
+    """Determine the user's LDAP based on the DN and attributes."""
+    if config.attrs_mapping["urn"] in attrs:
+        return attrs[config.attrs_mapping["urn"]][0].decode()
+    else:  # for backward compatiblity
```

Comment:
Let's add a log line / warning here that we could not find the configured attribute mapping, falling back to X. 

I'm a bit torn on this - we might want to simply skip the user instead of using attributes the user doesn't want to use.

Reply:
sure!

Reply:
Got it - thanks for explanation